svgelements/svgelements.py

```diff
@@ -3071,21 +3083,33 @@ def __str__(self):
         )
 
     def __repr__(self):
-        return "%s('%s')" % (self.__class__.__name__, str(self))
+        values = []
+        if self.x is not None:
+            values.append("x=%s" % Length.str(self.x))
+        if self.y is not None:
+            values.append("y=%s" % Length.str(self.y))
+        if self.width is not None:
+            values.append("width=%s" % Length.str(self.width))
+        if self.height is not None:
+            values.append("height=%s" % Length.str(self.height))
+        if self.preserveAspectRatio is not None:
+            values.append("%s=%s" % (SVG_ATTR_PRESERVEASPECTRATIO, self.preserveAspectRatio))
+        params = ", ".join(values)
+        return "Viewbox(%s)" % params
 
     def property_by_object(self, obj):
         self.x = obj.x
         self.y = obj.y
         self.width = obj.width
         self.height = obj.height
-        self.preserve_aspect_ratio = obj.preserve_aspect_ratio
+        self.preserveAspectRatio = obj.preserveAspectRatio
```

Comment:
preserveAspectRatio is snake rather than camel case by design. It's also can't be changed at this point since some other code in other projects using this might reference it.

Reply:
Agreed - will change back.